dashboard/src/actions/auth.ts

```diff
@@ -14,7 +14,16 @@ export const authenticationError = createAction("AUTHENTICATION_ERROR", resolve
   return (errorMsg: string) => resolve(errorMsg);
 });
 
-const allActions = [setAuthenticated, authenticating, authenticationError];
+export const setSessionExpired = createAction("SET_AUTHENTICATION_SESSION_EXPIRED");
+export const unsetSessionExpired = createAction("UNSET_AUTHENTICATION_SESSION_EXPIRED");
```

Comment:
why not combine this with `setSessionExpired` and give it a parameter of what to set? Similar to what we're doing for `setAuthenticated`.